src/net.cpp

```diff
@@ -3156,6 +3161,19 @@ uint64_t CConnman::CalculateKeyedNetGroup(const CAddress& address) const
     return GetDeterministicRandomizer(RANDOMIZER_ID_NETGROUP).Write(vchNetGroup.data(), vchNetGroup.size()).Finalize();
 }
 
+AddedNodeParams CConnman::GetAddedNodeParams(std::string added_node)
```

Comment:
```suggestion
AddedNodeParams CConnman::GetAddedNodeParams(std::string added_node) const
```

Reply:
This became obsolete, since GetAddedNodeParams was removed.